docs/querying/sql-functions.md

```diff
@@ -1077,11 +1077,42 @@ Calculates the base-10 of the numeric expression.
 
 ## LOOKUP
 
-`LOOKUP(<CHARACTER>, <CHARACTER>[, <CHARACTER>])`
+Look up `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. Returns `replaceMissingValueWith`, which defaults to `null`, when the `expr` is null or when the lookup does not contain a value for `expr`.
```

Comment:
Other examples use third-person singular form of the verb in simple present tense (a mouthful). "Looks up" doesn't sound great. Maybe "searches for" is a better choice? 
```suggestion
Searches for `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. Returns `replaceMissingValueWith`, which defaults to `null` if `expr` is null or if the lookup doesn't contain a value for `expr`.
```